courses/free-courses-hi.md

```diff
@@ -167,10 +167,9 @@
 
 * [2-D Arrays - Level 1](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFkOETg2Ybq-FMuJjkZSGeH) - Pepcoding
 * [All Data Structure Concepts \| Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefJNztGGoQC-59UhSwIaiIW3) - Saurabh Shukla
-* [All Sorting algorithms and Programs \| Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefJU-Sj__mljvrmJHZVKWbEm) - Saurabh Shukla
```

Comment:
This course is already present on LINE 55 (https://github.com/EbookFoundation/free-programming-books/pull/8540/files#diff-0bed9be5a9c29db386438a12b2e1abaac6945680b2e2ab9d07c0b58063e315adR55) of this file also there is mismatch in the URL i.e. on LINE 171 and 172 have swapped URL's compared to title and course on LINE 170 is already present on LINE 55.